kotlin-sample/app/src/main/res/layout/fragment_map.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why do we need `LinearLayout`, can we just use `ComposeView` as the root view?

Reply:
We can use ComposeView as the root view